test/runAll.unmocked.spec.js

```diff
@@ -358,6 +358,16 @@ index f80f875..1c5643c 100644
 
     // Remove lock file
     await fs.remove(`${cwd}/.git/index.lock`)
+
+    // Luckily there is a stash
+    expect(await execGit(['stash', 'list'])).toMatchInlineSnapshot(
```

Comment:
Question: do we plan to apply this automatically if we detect this case or we just inform users to do this manually?